src/main/java/com/playmonumenta/networkrelay/RemotePlayerManager.java

```diff
@@ -278,12 +289,15 @@ protected static void refreshLocalPlayer(Player player) {
 			mVisiblePlayers.add(remotePlayer.mUuid);
 		}
 		remotePlayer.broadcast();
+
+		RemotePlayerLoadedEvent remotePLE = new RemotePlayerLoadedEvent(remotePlayer);
```

Comment:
If I understand right, this event is supposed to fire every time player state changes and the player is loaded, even if they were already online - at least this is how it worked for remote players. We can adjust it if needed.